### ISSUE TITLE
Scroll problem on long notes

### ISSUE THREAD

**Issue by egesu** [khuttun/NotificationNotes#20]
I wrote a note and there was 4-5 paragraphs and 10 sentences in each paragraph. I saved it and open editing again. Then I could not scroll to the place I left writing in the end. When keyboard is shown, it is on top of the bottom lines so I cannot continue writing.

My device is Samsung Galaxy S3 and I use the app on CM11.

**Comment by khuttun** [comment]
Are you able to scroll the note if you hide the keyboard?

**Comment by egesu** [comment]
Yes. Then when I reveal the keyboard, it's on top of the bottom of the note.
